enterprise/api/v1/urls.py

```diff
@@ -4,19 +4,25 @@
 
 from rest_framework.routers import DefaultRouter
 
+from django.conf import settings
 from django.urls import re_path
 
 from enterprise.api.v1 import views
 
 router = DefaultRouter()
-router.register("enterprise_catalogs", views.EnterpriseCustomerCatalogViewSet, 'enterprise-catalogs')
+
+# Endpoints are deprecated. See https://github.com/openedx/public-engineering/issues/61
+# This endpoint is no longer used, so it will be removed soon
+if getattr(settings, 'ENABLE_DEPRECATED_API_ENDPOINTS', True):
```

Comment:
I think we should also ensure that we have toggle documentation for this setting so that we can keep track of it. 

Reference to documenting this toggle: https://edx.readthedocs.io/projects/edx-toggles/en/latest/how_to/documenting_new_feature_toggles.html